recipe/symfony.php

```diff
@@ -22,17 +26,29 @@
 
 // Assets
 set('assets', ['web/css', 'web/images', 'web/js']);
-// Default true - BC for Symfony < 3.0
-set('dump_assets', true);
+
+// Requires non symfony-core package `kriswallsmith/assetic` to be installed
+set('dump_assets', false);
```

Comment:
@robfrawley I think you should split this problem to a PR, because it is break BC. We will discuss about it more.